crates/rpc-types/src/eth/transaction/optimism.rs

```diff
@@ -0,0 +1,53 @@
+//! Misc Optimism-specific types
+use alloy_primitives::{B256, U128, U256, U64};
+use serde::{Deserialize, Serialize};
+
+use crate::other::OtherFields;
+
+/// Optimism specific transaction fields
+#[derive(Debug, Copy, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct OptimismTransactionFields {
```

Comment:
@mattsse prob follow-up refactor for `op-rpc-types`?

Reply:
yeah maybe we want them separate, but perhaps not actually worth it